tests/workspace-files.feature

```diff
@@ -0,0 +1,29 @@
+# Tests for the presence of files in the workspace
+
+Feature: Workspace files
+
+    As a researcher,
+    I want to make sure that my Snakemake workflow produces expected files,
+    and that the files fetched from the remote locations are not corrupted,
+    so that I can be sure that the workflow outputs are correct.
+
+    Scenario: The workspace contains all the relevant input files
+        When the workflow execution completes
+        Then the workspace should include "BuildFile.xml"
+        And the workspace should include "datasets/CMS_Run2011A_DoubleMu_AOD_12Oct2013-v1_10000_file_index.txt"
+        And the workspace should include "datasets/CMS_Run2011A_DoubleMu_AOD_12Oct2013-v1_10001_file_index.txt"
+        And the workspace should include "datasets/CMS_Run2011A_DoubleMu_AOD_12Oct2013-v1_20000_file_index.txt"
+        And the workspace should include "datasets/CMS_Run2011A_SingleMu_AOD_12Oct2013-v1_10000_file_index.txt"
+        And the workspace should include "datasets/CMS_Run2011A_SingleMu_AOD_12Oct2013-v1_10001_file_index.txt"
+        And the workspace should include "datasets/CMS_Run2011A_SingleMu_AOD_12Oct2013-v1_20000_file_index.txt"
+        And the workspace should include "datasets/CMS_Run2011A_SingleMu_AOD_12Oct2013-v1_20001_file_index.txt"
+        And the workspace should include "src/DimuonSpectrum2011.cc"
+        And the workspace should include "python/demoanalyzer_cfi.py"
+        And the workspace should include "demoanalyzer_cfg.py"
+
+    Scenario: The analysis produces the expected ROOT file
```

Comment:
Cosmetics: Perhaps call it "expected output files", because it's more important to look for "outputs" rather than in which format they are.